src/algebra/big_operators/fin.lean

```diff
@@ -28,4 +29,48 @@ lemma prod_filter_succ_lt {M : Type*} [comm_monoid M] {n : ℕ} (j : fin n) (v :
     ∏ j in univ.filter (λ i, j < i), v j.succ :=
 by rw [univ_filter_succ_lt, finset.prod_map, rel_embedding.coe_fn_to_embedding, coe_succ_embedding]
 
+@[to_additive]
+lemma prod_congr {M : Type*} [comm_monoid M] {a b : ℕ} (f : fin b → M) (h : a = b) :
+  ∏ (i : fin a), f (fin.cast h i) = ∏ (i : fin b), f i :=
+by { subst h, congr, ext, congr, ext, rw fin.coe_cast, }
+
+@[to_additive]
+lemma prod_split {M : Type*} [comm_monoid M] {a b : ℕ} (f : fin (a+b) → M) :
```

Comment:
This should be called `sum_univ_add` to match `sum_univ_succ`

Reply:
Thanks! I have renamed the lemma.